python/ray/node.py

```diff
@@ -639,10 +652,47 @@ def start_raylet(self, use_valgrind=False, use_profiler=False):
         assert ray_constants.PROCESS_TYPE_RAYLET not in self.all_processes
         self.all_processes[ray_constants.PROCESS_TYPE_RAYLET] = [process_info]
 
+    def get_job_redirected_log_file(self, worker_id, job_id):
+        """Determines (but does not create) logging files for workers to
+        redirect its output.
+
+        Args:
+            worker_id (bytes): A byte representation of the worker id.
+            job_id (bytes): A byte representation of the job id.
+
+        Returns:
+            (tuple) The stdout and stderr file names that the job should be
+        redirected to.
+        """
+        redirect_output = self._ray_params.redirect_output
+
+        if redirect_output is None:
+            # Make the default behavior match that of glog.
+            redirect_output = os.getenv("GLOG_logtostderr") != "1"
+
+        if not redirect_output:
+            return None, None
+
+        worker_stdout_file, worker_stderr_file = (self.get_log_file_names(
+            "worker-{}-{}".format(
+                ray.utils.binary_to_hex(worker_id),
+                ray.utils.binary_to_hex(job_id)),
+            unique=False))
+        return worker_stdout_file, worker_stderr_file
+
     def new_worker_redirected_log_file(self, worker_id):
-        """Create new logging files for workers to redirect its output."""
-        worker_stdout_file, worker_stderr_file = (
-            self.new_log_files("worker-" + ray.utils.binary_to_hex(worker_id)))
+        """Determines (but does not create) logging files for workers to
```

Comment:
Can we combine this with the above function? Have an optional current_job_id=None.